pandas/core/groupby/generic.py

```diff
@@ -760,6 +759,92 @@ def fillna(
         limit: int | None = None,
         downcast: dict | None = None,
     ) -> Series | None:
+        """
+        Fill NA/NaN values using the specified method within groups.
+
+        Parameters
+        ----------
+        value : scalar, dict, Series, or DataFrame
+            Value to use to fill holes (e.g. 0), alternately a
+            dict/Series/DataFrame of values specifying which value to use for
+            each index (for a Series) or column (for a DataFrame).  Values not
+            in the dict/Series/DataFrame will not be filled. This value cannot
+            be a list. Users wanting to use the ``value`` argument and not ``method``
+            should prefer :meth:`.Series.fillna` as this
+            will produce the same result and be more performant.
+        method : {{'bfill', 'ffill', None}}, default None
+            Method to use for filling holes. ``'ffill'`` will propagate
+            the last valid observation forward within a group.
+            ``'bfill'`` will use next valid observation to fill the gap.
+        axis : {0 or 'index', 1 or 'columns'}
+            Unused, only for compatibility with :meth:`DataFrameGroupBy.fillna`.
+        inplace : bool, default False
+            Broken. Do not set to True.
```

Comment:
For both DataFrameGroupBy and SeriesGroupBy, using inplace=True doesn't modify the source and returns  an empty DataFrame or a Series filled with `None` respectively. I plan to deprecate this in a followup.